### PR TITLE
Remove rocm5.2 warnings from builds

### PR DESCRIPTION
**Pull request by r-abishek**
Updates to remove rocm5.2 file-deprecation warnings from hip/host/ocl builds

### CODE REVIEW

File: utilities/rpp-performancetests/HIP_NEW/CMakeLists.txt [hunk 1]
```diff
@@ -1,17 +1,27 @@
 cmake_minimum_required(VERSION 3.0)
-project(Rpp_test)
+project(rpp_performancetests/HIP_NEW)
```
Comment (LakshmiKumar23) [inline utilities/rpp-performancetests/HIP_NEW/CMakeLists.txt:2]: @r-abishek Do we want to name this NEW? Similarly for the others? If the older ones are deprecated or removed maybe we can rename that and keep this as HIP?

Comment (r-abishek) [reply utilities/rpp-performancetests/HIP_NEW/CMakeLists.txt:2]: Yeah the NEW word has been in our pipeline to remove. It's present in the name of the folders too, but renaming the folders will cause git to interpret it as a re-creation of all files inside. Test suite renaming with some updates will be done as a separate PR

**Comment by LakshmiKumar23** [comment]
@r-abishek Seems like one more warning is seen from CI for ```cmake```. No warnings seen on ```make```
```
cmake -DBACKEND=OCL -DCMAKE_BUILD_TYPE=Release ../..

-- The C compiler identification is GNU 9.4.0

-- The CXX compiler identification is GNU 9.4.0

-- Detecting C compiler ABI info

-- Detecting C compiler ABI info - done

-- Check for working C compiler: /usr/bin/cc - skipped

-- Detecting C compile features

-- Detecting C compile features - done

-- Detecting CXX compiler ABI info

-- Detecting CXX compiler ABI info - done

-- Check for working CXX compiler: /usr/bin/c++ - skipped

-- Detecting CXX compile features

-- Detecting CXX compile features - done

-- Radeon Performance Primitives (RPP) Version -- 0.95

-- RPP install path set to -- /opt/rocm/rpp

-- CMake minimum version required: 3.5

-- CMake version found: 3.23.0-rc2

CMake Warning (dev) at /opt/cmake-3.23.0-rc2/share/cmake-3.23/Modules/FindPackageHandleStandardArgs.cmake:438 (message):

  The package name passed to `find_package_handle_standard_args` (OPENCL)

  does not match the name of the calling package (OpenCL).  This can lead to

  problems in calling code that expects `find_package` result variables

  (e.g., `_FOUND`) to follow a certain pattern.

Call Stack (most recent call first):

  cmake/FindOpenCL.cmake:52 (find_package_handle_standard_args)

  CMakeLists.txt:114 (find_package)

This warning is for project developers.  Use -Wno-dev to suppress it.



-- Found OPENCL: /opt/rocm/opencl/lib/libOpenCL.so  

-- ROCm OpenCL Found - Set CL_TARGET_OPENCL_VERSION=220

-- Added custom command for rpp_kernels

-- OpenCL kernels added!

-- OpenCL include directories -- /var/jenkins_home/workspace/ipclang_Short-GPUOpen_rpp_PR-116/gYwNxhPKM/rpp/src/include/cl/;/var/jenkins_home/workspace/ipclang_Short-GPUOpen_rpp_PR-116/gYwNxhPKM/rpp/src/include/common/;/opt/rocm/include/

-- amd_rpp set to build with GPU_SUPPORT using OpenCL backend

-- amd_rpp set to build with ENABLE_SIMD_INTRINSICS

-- Found Boost: /usr/lib/x86_64-linux-gnu/cmake/Boost-1.71.0/BoostConfig.cmake (found version "1.71.0") found components: filesystem system 

-- Found OpenMP_C: -fopenmp (found version "4.5") 

-- Found OpenMP_CXX: -fopenmp (found version "4.5") 

-- Found OpenMP: TRUE (found version "4.5")  

-- Configuring done

-- Generating done
```

**Review by LakshmiKumar23 — APPROVED** [review]

**Comment by r-abishek** [comment]
@LakshmiKumar23 Interesting, don't observe that in my build. Building on 5.2.
image

**Pull request merged**